ydb/docs/en/core/integrations/ingestion/toc-ingestion.yaml

```diff
@@ -0,0 +1,9 @@
+items:
+- name: Log records collection using FluentBit
+  href: fluent-bit.md
+- name: Logstash plugins
+  href: logstash.md
+- name: Importing from JDBC data sources
+  href: import-jdbc.md
+- name: Spring Data JDBC
+  href: spring-data-jdbc.md
```

Comment:
Afaik it's a more generic library than just for injection: let's create a new category called https://en.wikipedia.org/wiki/Object%E2%80%93relational_mapping out of Spring Data and Hibernate